index.html

```diff
@@ -265,26 +256,40 @@ <h2>Stop waiting.<br>Start building.</h2>
 
     <section class="contact" id="contact">
       <div class="container">
-        <h2>Contact
-          <i class="fa fa-heart"></i>
-          us!</h2>
-        <ul class="list-inline list-social">
-          <!--<li class="list-inline-item social-twitter">
-            <a href="#">
-              <i class="fa fa-twitter"></i>
-            </a>
-          </li>-->
-          <li class="list-inline-item social-github">
-            <a href="https://github.com/Overtorment/BlueWallet" target="_blank">
-              <i class="fa fa-github"></i>
-            </a>
-          </li>
-          <li class="list-inline-item social-facebook">
-            <a href="mailto:bluewallet at bluewallet.io">
-              <i class="fa fa-envelope"></i>               
-            </a>
-          </li>
-        </ul>
+        <h2>About</h2>
+        <p>We are <strong>Bitcoin</strong> enthusiasts focus on bringing the revolution to the people. 🔥<br />Reach out, we will appreciate your feedback.</p>
+        
+        <div class="row">
+          <div class="col-lg-4 people">
+            <img src="img/igor.png">
+            <h3><a href="https://twitter.com/overtorment" target="_blank">Igor Korsakov</a></h3>
+            <p>London</p>
+            <p>Engineering</p>
+          </div>
+          <div class="col-lg-4 people">
+            <img src="img/nuno.png">
+            <h3><a href="https://twitter.com/nvcoelho" target="_blank">Nuno Coelho</a></h3>
+            <p>Barcelona</p>
+            <p>Product & UX</p>
+          </div>
+          <div class="col-lg-4 people">
+            <img src="img/volodymyr.png">
+            <h3>Volodymyr Mishyn</h3>
+            <p>London</p>
+            <p>Strategy</p>
+          </div>
+        </div>
+    
+        <div class="row">
+          <div class="col-lg-12 reach">
+            <h3>Contact us</h3>
+            <div> <a href="mailto:bluewallet@bluewallet.io"><img src="img/reach.png"></a></div>
```

Comment:
making email an image makes no sense, email is still parse-able by spammers, but cant be copypasted by regular users